### DISCUSSION TITLE
Request aborted indicator on BackEnd

### DISCUSSION THREAD

**Discussion by mkalinski93** [nuxt/nuxt#25859] [Questions]
Hello,

I just wanted to know how I can get to know if the request has been canceled through FrontEnd.
My current project is forwarding the request to an external API and acts as an BackEnd-For-FrontEnd.

The useFetch request can be canceled with an AbortController - that works fine. But how can I get to know in the BackEnd-For-FrontEnd if the pending request has been canceled?

**Comment by Gonerman** [comment]
Hi, did you find a solution? @mkalinski93

**Reply by mkalinski93** [reply]
Unfortunately not

**Comment by mikesaintsg** [comment]
You would have to access the Nodejs Request object directly and check the readableAborted boolean

```javascript
export default defineEventHandler((event) => {
    event.node.req.readableAborted
})
```
